README.md

```diff
@@ -52,21 +52,64 @@ whenever commits are made to this repository and for all pull requests.
 
 ### Releasing a new version
 
-The repository uses the `git flow` framework for releasing versions. When the `develop`
-branch contains a version of the code to be released as a new version, `git flow release
-start x.y.z` is used to start a release branch. When that has passed testing, the
-command `git flow release finish x.y.z` creates a new tagged version on the `master`
-branch.
+All new package releases should be from the `master` branch, so the changes to `develop`
+have to be moved here. This is achieved using a `release` branch.
+
+```bash
+git branch release/x.y.z
+```
+
+This branch should be pushed to the remote repo to ensure that other developers have
+access to it.
+
+```bash
+git push --set-upstream origin release/x.y.z
+```
+
+This gives all developers an opportunity to fix problems they have noticed, prior to
+release. This can be done by making pull requests against the `release` branch. Once
+these changes have been made, documentation has been checked, and all tests pass, the
+final commit should bump the version using `poetry` so that the correct version is
+recorded in `pyproject.toml`.
+
+```bash
+poetry version x.y.z
+```
+
+The `release` branch now is ready to be merged with the `master` branch.
+
+```bash
+git switch master
+git merge release/x.y.z
+```
+
+A tag should be added marking the package version, and then the updated `master` branch
+should be pushed to the remote repository.
+
+```bash
+git tag x.y.z
+git push origin x.y.z
+```
+
+Finally, any changes added to the `release` branch should be merged back into `develop`.
+
+```bash
+git switch develop
+git merge release/x.y.z
+git push
+```
 
 Note that the `master` branch should _only_ be used for new releases.
 
-To publish a new version, first create the source distribution and a binary.
+Before the package is published it must first be built, this can be achieved using
+`poetry`.
 
-```{sh}
-# Create distribution
-python setup.py sdist bdist_wheel
+```bash
+poetry build
 ```
 
+This produces both a `sdist` source distribution, and a `wheel` compiled package.
+
 If the distribution is to be tested locally, then the following command can be used to
 install the source distribution:
```

Comment:
"Installed locally"? You'd only want to do this if you were wanting to install it to a python environment *outside* of a development environment managed by `poetry`. Clarify this.

Reply:
Ahh yes, this was from the previous README and wasn't sure whether to keep it or not. Probably can just go I guess